### ISSUE TITLE
setCancelConfiguration fails

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bdshadow/gwtupload#207]
```
https://mail.google.com/mail/u/0/?ui=2&view=btop&ver=42yqp1uuwq0f&q=gwtupload&qs
=true&search=query&th=143fec2dbd7b6563&qt=gwt.1.gwtupload.1.gwtuploaded.1.gwtupl
oading.1.gwtuploads.1.upload.1&cvid=1


```

Original issue reported on code.google.com by `manuel.carrasco.m` on 7 Jan 2015 at 9:21

**Comment by GoogleCodeExporter** [comment]
```
https://groups.google.com/forum/#!msg/gwtupload/ebw1sQfM7q0/5TBF3DPvriMJ
```

Original comment by `manuel.carrasco.m` on 8 Jan 2015 at 4:21

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 16eac6771a2b.
```

Original comment by `manuel.carrasco.m` on 13 Jan 2015 at 10:38
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `manuel.carrasco.m` on 13 Jan 2015 at 5:41
- Added labels: **Milestone-1.0.2**
